spynnaker/pyNN/models/neural_projections/connectors/kernel_connector.py

```diff
@@ -14,14 +14,14 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 import numpy
-from pyNN.random import RandomDistribution
-from .abstract_connector import AbstractConnector
-from spynnaker.pyNN.exceptions import SpynnakerException
 from spinn_utilities.overrides import overrides
 from data_specification.enums.data_type import DataType
 from spinn_front_end_common.utilities.constants import BYTES_PER_WORD
-from .abstract_generate_connector_on_machine \
-    import AbstractGenerateConnectorOnMachine, ConnectorIDs, PARAM_TYPE_KERNEL
+from pyNN.random import RandomDistribution
```

Comment:
I thought it was standard procedure to put "third party" froms / imports before SpiNNaker ones?

Reply:
Fixed